src/connection/mod.rs

```diff
@@ -75,7 +84,8 @@ pub fn create(s: &str, dt: DeviceType, sc: Security, p: &str) -> Result<Connecti
 /// # Examples
 ///
 /// ```
-/// let mut connections = network_manager::connection::list().unwrap();
+/// let manager = network_manager::manager::NetworkManager::new();
```

Comment:
Do you think its possible to flatten the structure a little? `network_manager::manager::NetworkManager::new();` is quite verbose.